### PR TITLE
Integration tests using goth

### PR DESCRIPTION
**Pull request by azawlocki**
Resolves: #147

### CODE REVIEW

File: tests/goth/pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+]
+repository = "https://github.com/golemfactory/yajsapi"
+documentation = "https://handbook.golem.network/yajsapi/"
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "^3.8.0"
+pytest = "^5.4.3"
+pytest-asyncio = "^0.12.0"
+goth = {git = "https://github.com/golemfactory/goth.git", branch = "master"}
```
Comment (azawlocki) [inline tests/goth/pyproject.toml:25]: Meanwhile `goth` got published on `pypi` so we can just say `goth = "^0.1.0"` here.

File: tests/goth/pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+[tool.poe.tasks]
+goth-assets = "python -m goth create-assets tests/goth/assets"
+goth-tests = "pytest -svx tests/goth"
```
Comment (azawlocki) [inline tests/goth/pyproject.toml:29]: At the moment those aliases are not used. We can either use them in the workflow file, by replacing e.g. `poetry run python -m goth create-assets assets` with `poetry run poe goth-assets` or remove the whole `tool.poe.tasks` section from `pyproject.toml`.

File: tests/goth/pyproject.toml (hunk 1 above)
Comment (filipgolem) [reply tests/goth/pyproject.toml:25]: Done.

File: tests/goth/pyproject.toml (hunk 2 above)
Comment (filipgolem) [reply tests/goth/pyproject.toml]: Removed.

**Review by filipgolem — APPROVED** [review]

**Pull request merged**